packages/tictactoe/src/redux/game/__tests__/player-a.test.ts

```diff
@@ -10,8 +10,8 @@ import {
 } from "../../../core";
 import * as actions from "../actions";
 import * as state from "../state";
-import BN from "bn.js";
-import bnToHex from "../../../utils/bnToHex";
+// import BN from "bn.js";
```

Comment:
Any reason not to remove this commented out code? Generally, commented out code reduces readability

Reply:
👍